### PR TITLE
about us page started

### PR DESCRIPTION
**Pull request by Zunuoh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import "./App.css";
 import "bootstrap/dist/css/bootstrap.min.css";
 import Homescreen from "./components/homescreen/Homescreen";
 import ContactusScreen from "./components/contact/ContactusScreen";
+import Aboutus from "./components/aboutus/Aboutus";
```
Comment (claeusdev) [inline src/App.js:5]: It's always good to name your top-level components using `CamelCase`.

Also to be able to separate concerns, ie: grouping things related to their intended purpose. Components that are "pages" should be placed under a top-level directory eg. `pages/AboutUs`. `components` should be used for things that are reusable and I don't think you're going to be reusing the about page a whole lot.

The benefit of this is that you're going easily figure out where things are.

File: src/components/aboutus/Aboutus.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      <div style={{flex:1, marginLeft:90}}>
+        <div className='aboutusNumbers'>200+</div>
+        <div className='aboutusHeader'>5-star reviews</div>
+        <div style={{fontSize:15}}>We're proud of our 5-star rating with over 200 reviews.</div>
+      </div>
+    </div>
+    
+    <div className='aboutHeader'>From our team</div>
+
+    <div style ={{}}>
```
Comment (claeusdev) [inline src/components/aboutus/Aboutus.js:40]: Remember to always remove empty braces from code.

**Review by claeusdev — CHANGES_REQUESTED** [review]
Overall this looks good, just a few comments:

1. You should endeavor to use `.js` extensions for pure javascript functionality and then `jsx` for your components. Easy for anyone who picks up your work to know the difference between the two. see [here](https://github.com/airbnb/javascript#references) for example best practices.
2. Looking at your `Aboutus` components, you could really break it down into much smaller bits that you can compose to build bigger components/pages. 
3. [This](https://github.com/airbnb/javascript/tree/master/react#airbnb-reactjsx-style-guide) is one of the places I've used to get some best practices and also help me understand some of the decisions. 
4. Finally, please always remember to open PRs like this and make them small also like this. This way it'll be very easy to review and give some quality feedback.

**Review by claeusdev — APPROVED** [review]
LGTM .. Looks great.

**Comment by Zunuoh** [comment]
Thanks

**Pull request merged**